BEIMA.Client/src/App.js

```diff
@@ -1,23 +1,30 @@
 import './App.css';
+import { useState } from "react"
 import {
   BrowserRouter,
   Routes,
   Navigate,
   Route,
-  Outlet,
-  useLocation
+  Outlet
 } from "react-router-dom";
 import HomePage from './pages/Home/HomePage';
 import HelpPage from './pages/Help/HelpPage';
+import AddDevicePage from './pages/Devices/AddDevicePage';
+import AddDeviceTypePage from './pages/DeviceTypes/AddDeviceTypePage';
 import NavBar from './shared/NavBar';
 import PageTitle from './shared/PageTitle';
+import 'bootstrap/dist/css/bootstrap.min.css';
 
 function App() {
   return (
     <BrowserRouter>
       <Routes>
         <Route path="/" element={<AppLayout />}>
           <Route index element={<HomePage />} />
+
+          <Route path="addDevice" element={<AddDevicePage/>}/>
+          <Route path="addDeviceType" element={<AddDeviceTypePage/>}/>
+
           <Route path="Help" element={<HelpPage />} />
```

Comment:
Is there a reason for the extra newlines here?

Reply:
Not really, got rid of them now